diagram.md

```diff
@@ -2,240 +2,284 @@
+|        |            | Audio        | SamplingFrequency |
+|        |            |              |-------------------|
+|        |            |              | Channels          |
+|        |            |              |-------------------|
+|        |            |              | BitDpeth          |
+|--------------------------------------------------------|
+
+```
+Figure: Representation of the `Tracks Element` and a selection of its `Descendant Elements`.
+
+The `Chapters Element` section lists all of the Chapters. Chapters are a way to set predefined points to jump to in video or audio.
+
+```
++----------------------------------------------+
+| Chatpers | EditionEntry | EditionUID         |
+|          |              |--------------------|
+|          |              | EditionFlagHidden  |
+|          |              |--------------------|
+|          |              | EditionFlagDefault |
+|          |              |--------------------|
+|          |              | EditionFlagOrdered |
+|          |              |--------------------|-------------------+
+|          |              | ChapterAtom        | ChapterUID        |
+|          |              |                    |-------------------|
+|          |              |                    | ChapterStringUID  |
+|          |              |                    |-------------------|
+|          |              |                    | ChapterTimeStart  |
+|          |              |                    |-------------------|
+|          |              |                    | ChapterTimeEnd    |
+|          |              |                    |-------------------|
+|          |              |                    | ChapterFlagHidden |
+|          |              |                    |-------------------|--------------+
+|          |              |                    | ChapterDisplay    | ChapString   |
+|          |              |                    |                   |--------------| 
+|          |              |                    |                   | ChapLanguage | 
++---------------------------------------------------------------------------------+
+```
+Figure: Representation of the `Chapters Element` and a selection of its `Descendant Elements`.
+
+The `Cluster Elements` contain all of the video frames and audio for each track. In a given Matroska file, there are usually many `Cluster Elements`. The Clusters help to break up the `SimpleBlock` or `BlockGroup Elements` and help with seeking and error protection. There is no set limit to how much data a Cluster can contain, or how much time they can span, but a limit at 5 seconds or 5 megabytes is RECOMMENDED. Every Cluster contains a timecode, usually the timecode that the first Block in the Cluster SHOULD be played back, but it doesn't have to be. Then there are one or more (usually many more) `BlockGroups` or `SimpleBlocks` in each Cluster. A BlockGroup can contain a Block of data, and any information relating directly to that Block.
```

Comment:
The recommendation is the same before and after the pr, so I recommend that the issue not block the pr unless it can be resolved quickly, else let's move to an issue.